TEveEventDisplay/src/TEveMu2eDataInterface.cc

```diff
@@ -376,43 +377,54 @@ template <typename L> void maxminCRV(L data, double &max, double &min){
       TXYMgr->ImportElements(fTrackList2D, scene1); 
       TRZMgr->ImportElements(fTrackList2D, scene2); 
       if(seedcol!=0){  
-        for(unsigned int k = 0; k < seedcol->size(); k = k + 20){
+        for(unsigned int k = 0; k < seedcol->size(); k = k + 20){   
+       
           KalSeed kseed = (*seedcol)[k];
-          TEveMu2eCustomHelix *line = new TEveMu2eCustomHelix();
-          TEveMu2eCustomHelix *line_twoD = new TEveMu2eCustomHelix();
-          line->fKalSeed = kseed;
-          line->SetSeedInfo(kseed);
+          const std::vector<mu2e::KalSegment> &segments = kseed.segments();
+      size_t nSegments=segments.size();
+      if(nSegments==0) continue;
+      const mu2e::KalSegment &segmentFirst = kseed.segments().front();
+      const mu2e::KalSegment &segmentLast = kseed.segments().back();
+      double fltLMin=segmentFirst.fmin();
+      double fltLMax=segmentLast.fmax();
+    
+      TEveMu2eCustomHelix *line = new TEveMu2eCustomHelix();
+      TEveMu2eCustomHelix *line_twoD = new TEveMu2eCustomHelix();
```

Comment:
This is for Sophie:  I see that the line and line_twoD objects are passed to collections of type TEveElementList.  Does that class take ownership and look after deletion?

Reply:
Ah, good point, I've never officially checked but assumed once out of scope it is gone....I need to spend some time debugging anyway. Once Namitha's work is merged I'll run some checks